### PR TITLE
190819-wang-gang-chu-li-zhu-shu

### PR DESCRIPTION
**Pull request by GUThe**
#83

### CODE REVIEW

File: subtitles/wang-gang/190819-huanong‘s bamboo rat [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+Tell gradpa what this is
+
+B 竹鼠啊~~
+Rhizomys~~
+
+A 猪鼠啊？
+"Pig Rat"? (homophnic to Rhizomys)
+
+看着爷爷说
+Look at grandpa the speak
```
Comment (GUThe) [inline subtitles/wang-gang/190819-huanong‘s bamboo rat]:
```suggestion
Look at grandpa then speak
```

File: subtitles/wang-gang/190819-huanong‘s bamboo rat [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+Hahahah look at its expressions
+
+这个竹鼠就是我新认的兄弟送给我的
+My new sworn bro gifted the Rhizomys to me
+
+我跟他念了一年多终于送给我两只竹鼠
+I had been nagging him for more than a year untill he finally sent me two
+
+他说我最近人消瘦了
+He said I've lost weight lately
```
Comment (GUThe) [inline subtitles/wang-gang/190819-huanong‘s bamboo rat]:
```suggestion
He said I've lost some weight recently 
```

File: subtitles/wang-gang/190819-huanong‘s bamboo rat [hunk 3]
```diff
@@ -0,0 +1,262 @@
...
+Sure~~
+
+这个比较瘦， （笼子里）这个比较肥
+This one is relatively thin, the one in the cage is heavier
+
+B 它跑的很快
+It runs fast
+
+A 这个怕周身都是肉
+Dat is big hunks of flesh!
```
Comment (GUThe) [inline subtitles/wang-gang/190819-huanong‘s bamboo rat]:
```suggestion
Dat is big hunks of flesh all-around the body！
```

File: subtitles/wang-gang/190819-huanong‘s bamboo rat [hunk 4]
```diff
@@ -0,0 +1,262 @@
...
+Dady's gonna take the another one out
+
+唉哟这个好重哦
+Oh, this one is far heavier
+
+这个有三斤多
+It's more than 1.5 kg
+
+小心哦漆二娃来了的哦
+Take care, (any nickname that means "two"?)
```
Comment (GUThe) [inline subtitles/wang-gang/190819-huanong‘s bamboo rat]:
```suggestion
Take care, (any nickname that means "two"?) is around
```

File: subtitles/wang-gang/190819-huanong‘s bamboo rat [hunk 5]
```diff
@@ -0,0 +1,262 @@
...
+Oh, this one is far heavier
+
+这个有三斤多
+It's more than 1.5 kg
+
+小心哦漆二娃来了的哦
+Take care, (any nickname that means "two"?) is around
+
+B 嗯跟地滚滚一个样（鼹鼠）
+It looks exactly like mole
```
Comment (GUThe) [inline subtitles/wang-gang/190819-huanong‘s bamboo rat]:
```suggestion
It looks exactly like a mole
```

File: subtitles/wang-gang/190819-huanong‘s bamboo rat [hunk 6]
```diff
@@ -0,0 +1,262 @@
...
+Because we might need to feed the Rhizomys...
+
+养几天再弄来烹饪
+...for several days before proceeding to cook them
+
+这几天就砍竹子来喂它
+We'll just feed the with bamboos the next few days
+
+你在干嘛啊沺沺
+What are doing Tiantian?
```
Comment (GUThe) [inline subtitles/wang-gang/190819-huanong‘s bamboo rat]:
```suggestion
What are you doing Tiantian?
```

**Comment by GUThe** [comment]
明后天我都没时间，review完有人能帮忙上传一下下吗？拜托了~
Thanks in advance~

File: subtitles/wang-gang/190819-huanongs-bamboo-rat [hunk 7]
```diff
@@ -0,0 +1,262 @@
...
+I will process the Rhizomys..
+
+来让它体现出来
+...with my heart and the best technics 
+
+也不枉他对我的一番好意
+In my dearest respose to his kind act
+
+现在这么一说他们两个很紧张在里面
+They becomes very nervouse when i'm talking
```
Comment (immoonancient) [inline subtitles/wang-gang/190819-huanongs-bamboo-rat]:
```suggestion
They get very nervous when i'm talking
```

File: subtitles/wang-gang/190819-huanongs-bamboo-rat [hunk 8]
```diff
@@ -0,0 +1,262 @@
...
+
+跑来跑去的
+Running around...
+
+别怕啊
+Don't be scared
+
+哦对了兄弟说带你们过来是来旅游的
+Also my bro told me you are here for traveling purpose
```
Comment (immoonancient) [inline subtitles/wang-gang/190819-huanongs-bamboo-rat]:
```suggestion
By the way, my bro told me you are here for traveling purpose
```

File: subtitles/wang-gang/190819-huanongs-bamboo-rat [hunk 9]
```diff
@@ -0,0 +1,262 @@
...
+We have to get some bamboos to feed you
+
+四伯爷那个斧子呢
+Where's the uncle's axe
+
+因为这个竹鼠可能要养几天
+Because we might need to feed the Rhizomys...
+
+养几天再弄来烹饪
+...for several days before proceeding to cook them
```
Comment (immoonancient) [inline subtitles/wang-gang/190819-huanongs-bamboo-rat]:
```suggestion
...for several days before cooking them
```

File: subtitles/wang-gang/190819-huanongs-bamboo-rat [hunk 10]
```diff
@@ -0,0 +1,262 @@
...
+
+这几天就砍竹子来喂它
+We'll just feed the with bamboos the next few days
```
Comment (immoonancient) [inline subtitles/wang-gang/190819-huanongs-bamboo-rat]:
```suggestion
We'll just feed them with bamboos for the next few days
```

File: subtitles/wang-gang/190819-huanongs-bamboo-rat [hunk 11]
```diff
@@ -0,0 +1,262 @@
...
+It wants to get out babe
+
+要不要让它出来
+You want to let it out?
+
+B 要
+Yes!
+
+现在天气晚了，明天再放他出来好不好
+It's late today, will you let them out tomorrow?
```
Comment (immoonancient) [inline subtitles/wang-gang/190819-huanongs-bamboo-rat]:
```suggestion
It's late today, how about letting them out tomorrow?
```

File: subtitles/wang-gang/190819-huanongs-bamboo-rat [hunk 12]
```diff
@@ -0,0 +1,262 @@
...
+Alright it should be enough bamboos for them to enjoy
+
+这里两只竹鼠
+There are two Rhizomys here
+
+一只按照我的想法来烧
+One will be cooked with my method
+
+还有一只看大家有没有什么意见
+The other one will be cooked according to y'all's ideas
```
Comment (immoonancient) [inline subtitles/wang-gang/190819-huanongs-bamboo-rat]:
```suggestion
The other one will be cooked according to your ideas 
```
不要用 "y'all"，这个俚语词的地域/种族关联太强，主要是南方和黑人在用
https://en.wiktionary.org/wiki/y%27all
> (chiefly Southern US, African American Vernacular) Plural form of you.

File: subtitles/wang-gang/190819-huanongs-bamboo-rat [hunk 13]
```diff
@@ -0,0 +1,262 @@
...
+There are two Rhizomys here
+
+一只按照我的想法来烧
+One will be cooked with my method
+
+还有一只看大家有没有什么意见
+The other one will be cooked according to y'all's ideas 
+
+有没有你们最好的烹饪方法
+What's your best culinary methods?
```
Comment (immoonancient) [inline subtitles/wang-gang/190819-huanongs-bamboo-rat]:
```suggestion
What's your best recipe?
```

File: subtitles/wang-gang/190819-huanongs-bamboo-rat [hunk 14]
```diff
@@ -0,0 +1,262 @@
...
+
+告诉我我来烹饪他
+Tell me and I'll cook it in your way
+
+我来把它做成美食分享给大家
+I'll make it into delicious food and share to you
```
Comment (immoonancient) [inline subtitles/wang-gang/190819-huanongs-bamboo-rat]:
```suggestion
I'll make it into delicious food and share with you
```

File: subtitles/wang-gang/190819-huanongs-bamboo-rat [hunk 15]
```diff
@@ -0,0 +1,263 @@
...
+Don't be scared
+
+哦对了兄弟说带你们过来是来旅游的
+By the way, my bro told me you are here for traveling purpose
+
+我们这儿也得给你弄个竹子吃一下
+We have to get some bamboos to feed you
+
+四伯爷那个斧子呢
+Where's the uncle's axe
```
Comment (immoonancient) [inline subtitles/wang-gang/190819-huanongs-bamboo-rat]:
```suggestion
Where's the axe, uncle?
```

File: subtitles/wang-gang/190819-huanongs-bamboo-rat [hunk 16]
```diff
@@ -0,0 +1,263 @@
...
+Okay put the bamboo rats back
+
+这个竹鼠很不错
+The bamboo rats are quite good
+
+还是很感谢我这个刘苏良兄弟
+Kudos to my bro Liu Suliang (Huanong Brother)
+
+虽然说我这个兄弟从不吃亏
+My bro's famous for never taking any loses
```
Comment (immoonancient) [inline subtitles/wang-gang/190819-huanongs-bamboo-rat]:
```suggestion
My bro's famous for never taking any loss
```

File: subtitles/wang-gang/190819-huanongs-bamboo-rat [hunk 17]
```diff
@@ -0,0 +1,263 @@
...
+yet he still asked someone to bring me a pair...
+
+还托人从那么远拉过来
+... and from quite a distance
+
+非常不容易
+Not an easy task
+
+当然这个的话也不会辜负他的一番美意
+But I will definately not let him down
```
Comment (immoonancient) [inline subtitles/wang-gang/190819-huanongs-bamboo-rat]:
```suggestion
But I will definitely not let him down
```

**Pull request merged**